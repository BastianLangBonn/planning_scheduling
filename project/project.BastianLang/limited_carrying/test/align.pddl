(define (problem problem1)
	(:domain domain)
	(:objects 	s - s1
			o1 - o1
			o2 - o2
			r - robot
			c1 c2 - capacity)
	(:init 	(at_location r s)
		(at_location o1 s)
		(grapped r o2)
		(not(part_of_line o2))
		(= (distance s s) 0)
		(preceding_capacities c1 c2)
		(robot_carries r c2)
	)
	(:goal
		(and 
		(line o1 o2 s) 
		(gripper_free r) 
		(not(grapped r o2)) 
		(at_location o2 s) 
		(end_of_line o2) 
		(part_of_line o1) 
		(part_of_line o2)
		(not(robot_carries r c2))
		(robot_carries r c1)
		)
	)
)
