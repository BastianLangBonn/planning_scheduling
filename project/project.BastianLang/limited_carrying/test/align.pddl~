(define (problem problem1)
	(:domain domain)
	(:objects 	s - s1
			o1 - o1
			o2 - o2
			r - robot)
	(:init 	(at_location r s)
		(at_location o1 s)
		(grapped r o2)
		(not(part_of_line o2))
		(= (distance s s) 0)
	)
	(:goal
		(and (line o1 o2 s) (gripper_free r) (not(grapped r o2)) (at_location o2 s) (end_of_line o2) (part_of_line o1) (part_of_line o2))
	)
)
