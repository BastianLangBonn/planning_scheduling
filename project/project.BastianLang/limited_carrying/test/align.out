(align_with_object r o1 o2 s c1 c2)
